internal/common/page.go

```diff
@@ -101,7 +101,7 @@ func (p *Page) LeafPageElements() []leafPageElement {
 	}
 	var elems []leafPageElement
 	data := UnsafeAdd(unsafe.Pointer(p), unsafe.Sizeof(*p))
-	UnsafeSlice(unsafe.Pointer(&elems), data, int(p.count))
+	elems = unsafe.Slice((*leafPageElement)(data), int(p.count))
```

Comment:
ditto, remove line 102. Same below
```suggestion
	elems := unsafe.Slice((*leafPageElement)(data), int(p.count))
```